hmpps-prisoner-search-indexer/src/test/resources/application-test.yml

```diff
@@ -47,27 +47,22 @@ hmpps.sqs:
       queueName: ${random.uuid}
       dlqName: ${random.uuid}
       dlqMaxReceiveCount: 1
-      propagateTracing: false
```

Comment:
This was a previous workaround to fix tests - not needed now because the bug is fixed